include/revng/PipelineC/Tracing/Trace.h

```diff
@@ -138,6 +138,8 @@ struct RunTraceOptions {
   // Instead of using a temporary directory, the first invocation will use
   // these directory instead and subsequent ones will abort
   std::string ResumeDirectory;
+  // Is enabled, the manager will save after every analysis is run
```

Comment:
1. `s/Is/If/`
2. When you document a field in a `struct` use Doxygen (`///`)

Reply:
1. Ok
2. Ok